gpiozero/spi_devices.py

```diff
@@ -92,9 +131,9 @@ def _read(self):
     def value(self):
         """
         The current value read from the device, scaled to a value between 0 and
-        1.
+        1 (or -1 to +1 for certain devices operating in differential mode).
```

Comment:
Ahhh ... good point. And a property which advertises the range of `value` ... that's a very interesting idea. That needs a little thought as to how it handles edge cases like `None` (in `Servo`) but it's definitely worth exploring. I very much like the idea of the `source` property throwing a warning when connecting to a `values` which has a larger range than it can handle.